src/applications/mhv-landing-page/components/NavCard.jsx

```diff
@@ -10,18 +10,16 @@ const NavCard = ({ icon = null, title, links }) => {
         href={href}
         aria-label={ariaLabel}
         onClick={() => {
-          const header =
-            typeof text === 'object' && text !== null ? 'Inbox' : text;
-          const sectionHeader =
-            typeof text === 'object' && text !== null ? 'Messages' : title;
           recordEvent({
             event: 'nav-linkslist',
-            'links-list-header': header,
-            'links-list-section-header': sectionHeader,
+            'links-list-header': text,
+            'links-list-section-header': title,
           });
         }}
       >
-        {text}
+        <span className={ariaLabel?.includes('unread') ? 'indicator' : ''}>
```

Comment:
Does `indicator` need to be `mhv-c-indicator`, now?

Reply:
yes, good catch